tests/test_quantity.py

```diff
@@ -0,0 +1,560 @@
+    conversion_output = cr.to("m^-3 s^-1")
+    assert conversion_output.__float__() == pytest.approx(35.3147, 0.001)
+
+
+def test_area_inverse():
+    in_sq_m = q.Quantity(1, "m^-2")
+    conversion_output = in_sq_m.to("cm^-2")
+    assert conversion_output.__float__() == pytest.approx(0.0001)
+
+
+def test_area():
+    in_sq_m = q.Quantity(1, "m^2")
+    conversion_output = in_sq_m.to("in^2")
+    assert conversion_output.__float__() == pytest.approx(1550, 0.1)
+
+
+def test_angular_velocity():
+    degps = q.Quantity(1, "deg/s")
+    conversion_output = degps.to("rad/s")
+    assert conversion_output.__float__() == pytest.approx(0.01745, 0.001)
+
+
+def test_hz_hertz():
+    f = q.Quantity(1, "Hz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("hertz")
+
+
+def test_hz_rad():
+    f = q.Quantity(1, "Hz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("rad/s")
+
+
+def test_hz_radian():
+    f = q.Quantity(1, "Hz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("radian/s")
+
+
+def test_hz_rpm():
+    f = q.Quantity(1, "Hz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("rpm")
+
+
+def test_hz_rps():
+    f = q.Quantity(1, "Hz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("rps")
+
+
+def test_hz_cps():
+    f = q.Quantity(1, "Hz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("cps")
+
+
+def test_hertz_hz():
+    f = q.Quantity(1, "hertz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("Hz")
+
+
+def test_hertz_rad():
+    f = q.Quantity(1, "hertz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("rad/s")
+
+
+def test_hertz_radian():
+    f = q.Quantity(1, "hertz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("radian/s")
+
+
+def test_hertz_rpm():
+    f = q.Quantity(1, "hertz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("rpm")
+
+
+def test_hertz_rps():
+    f = q.Quantity(1, "hertz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("rps")
+
+
+def test_hertz_cps():
+    f = q.Quantity(1, "hertz")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("cps")
+
+
+def test_rad_hz():
+    f = q.Quantity(1, "rad/s")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("Hz")
+
+
+def test_rad_hertz():
+    f = q.Quantity(1, "rad/s")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("hertz")
+
+
+def test_rad_radian():
+    f = q.Quantity(1, "rad/s")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("radian/s")
+
+
+def test_rad_rpm():
+    f = q.Quantity(1, "rad/s")
+    with pytest.raises(Exception) as e_info:
+        conversion_output = f.to("rpm")
```

Comment:
Regarding my question in the other file, I never noticed this test before.  Why is this raising an Exception?

Reply:
It's raising exception because it is disallowed right now.

Reply:
Right, ok, but I'm asking what the rationale is to disallow it.

Reply:
Yes, it should be allowed but wrong pair of ```rad/s``` and ```rpm``` is formed due to considering all flat list permutations.

Reply:
This test case is removed now.